### PR TITLE
Run eslint

### PR DESCRIPTION
**Pull request by alyssamui**
Running eslint on the whole repo

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ gatsby-types.d.ts
 package.json
 .eslintrc.json
 .vscode
-tsconfig.json
\ No newline at end of file
+tsconfig.json
+README.me
```
Comment (surbhi-gulati) [inline .eslintignore:9]: Why README.me?

**Review by surbhi-gulati — APPROVED** [review]
Thanks Alyssa!

File: .eslintignore (hunk 1 above)
Comment (alyssamui) [reply .eslintignore:9]: i was getting eslint errors on the README and some other types of files, but when i committed it, it seemed to be fine?? so honestly that isn't necessary

**Review by mike10911 — APPROVED** [review]
THANK YOUUU. LGTM

**Pull request merged**